etabotsite/etabotapp/misc_utils/test_djangoindexhtml_mod.py

```diff
@@ -0,0 +1,26 @@
+import unittest
```

Comment:
Please structure the tests in the following structure:

```
etabotapp:
├── tests
│   ├── resources
│   │   ├── test_index.html
│   │   └── test_index_django_template.html
│   └── test_django_index_html.py
```

Reply:
looks good